search_params.py

```diff
@@ -36,26 +38,53 @@ def step(self):
         )
         logging.info(f'Run name: {self.config.run_name}')
 
+        checkpoint_dir = os.path.join(
+            self.config.result_dir, self.config.run_name)
+        best_checkpoint_path = os.path.join(checkpoint_dir, 'best_model.ckpt')
+        last_checkpoint_path = os.path.join(
+            checkpoint_dir, 'lastest_model.ckpt')
```

Comment:
Why we use `.ckpt` instead of `.pt` here? Is this a fix type of `load_from_checkpoint`?

Reply:
The extension name doesn't matters here. Since we are saving and loading checkpoint here, `.ckpt` is more semantic meaningful for my taste. Also, PL tutorial use `.ckpt` throughout the guide.

Reply:
I have the same question after reading that guide. What would you suggest?

Reply:
I will keep it that way